src/circuit/ecc.rs

```diff
@@ -183,6 +184,7 @@ impl<E: JubjubEngine> EdwardsPoint<E> {
             }
         )?;
 
+        // Add the curve constraints for (x,y)
```

Comment:
Comment about the comments: add "." to the end of sentences. I find it is less distracting (especially but not only when another sentence follows).

Reply:
Umm will do for the long explanations, for these in code one liners, Sean doesn't, so would be inconsistent